linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -228,7 +229,10 @@ case class HttpConfig(
   var service: Option[HttpSvc] = None
 
   private val streaming = streamingEnabled -> streamAfterContentLengthKB match {
-    case (Some(true), None) => hparam.Streaming(true)
+    case (Some(true), None) => hparam.Streaming(5.kilobytes)
+    case (None, None) => hparam.Streaming(5.kilobytes)
+    case (Some(false), None) => hparam.Streaming(false)
+    case (Some(false), Some(_)) => throw ConflictingStreamingOptions(_label.getOrElse("unknown"))  //tried label property, but it NPE's in test
```

Comment:
If you are talking about `HttpConfigTest`, it does not NPE for me if use the `def label`

Reply:
Strange. It did once. Now it's happy.